lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
Hi @PlasmaPower, the error is that you are not providing the first argument, `statusCode`, to the `writeHead` method. You code needs to be the following:
```js
try {
  this.res.writeHead(this.res.statusCode);
} catch (e) {
  console.error(e.stack);
}
```